test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/ReleaseCEnabledControllerSpec.scala

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eusubsidycompliancefrontend.controllers
+
+import com.typesafe.config.ConfigFactory
+import org.jsoup.Jsoup
+import org.scalatest.concurrent.ScalaFutures
+import play.api.Configuration
+import play.api.inject.bind
+import play.api.inject.guice.GuiceableModule
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import uk.gov.hmrc.auth.core.AuthConnector
+import uk.gov.hmrc.eusubsidycompliancefrontend.journeys.SubsidyJourney
+import uk.gov.hmrc.eusubsidycompliancefrontend.persistence.Store
+import uk.gov.hmrc.eusubsidycompliancefrontend.services._
+import uk.gov.hmrc.eusubsidycompliancefrontend.syntax.FutureSyntax.FutureOps
+import uk.gov.hmrc.eusubsidycompliancefrontend.test.CommonTestData._
+import uk.gov.hmrc.eusubsidycompliancefrontend.util.TimeProvider
+
+import scala.concurrent.Future
+
+class ReleaseCEnabledControllerSpec
```

Comment:
Can you please rename this to include the name of the class being tested: e.g. ReleaseCEnabledSubsidyControllerSpec